handbook/company/product-groups.md

```diff
@@ -14,13 +14,14 @@ At Fleet, [anyone can contribute](https://fleetdm.com/handbook/company#openness)
 
 ## Current product groups
 
-| Product group             | Goal _(value for customers and/or community)_                       |
-|:--------------------------|:--------------------------------------------------------------------|
-| [MDM](#mdm-group)                                       | Reach maturity in the "MDM" product category.
-| [Customer experience (CX)](#customer-experience-group)  | Make customers happier and more successful.
-| [Infrastructure](#infrastructure-group)                 | Provide and support reliable and secure infrastructure.
-| [Website](#website-group)                               | Make the website wonderful.
-
+| Product group             | Goal _(value for customers and/or community)_                       | Capacity\* |
+|:--------------------------|:--------------------------------------------------------------------|:-----------------|
+| [MDM](#mdm-group)                                       | Reach maturity in the "MDM" product category. | TODO     |
+| [Customer experience (CX)](#customer-experience-group)  | Make customers happier and more successful.   | TODO     |
+| [Infrastructure](#infrastructure-group)                 | Provide and support reliable and secure infrastructure. | TODO |
+| [Website](#website-group)                               | Make the website wonderful.                   | TODO     |
```

Comment:
We'll need these capacities filled in.  This is the usual capacity if everyone is there (don't count people who are still onboarding or not hired yet)

Reply:
48 for website group (they count points differently)